subprojects/core/src/main/java/org/gradle/initialization/buildsrc/BuildSrcBuildListenerFactory.java

```diff
@@ -32,17 +32,12 @@
 
     private static final String DEFAULT_BUILD_SOURCE_SCRIPT_RESOURCE = "defaultBuildSourceScript.txt";
 
-    Listener create(boolean rebuild) {
-        return new Listener(rebuild);
+    Listener create() {
+        return new Listener();
```

Comment:
I'm not sure if we can remove the previous "clean everything" behavior for buildSrc because we're not fixing all possible inconsistencies yet. So someone could have a custom plugin they use in buildSrc that will now be broken without a clean.